### DISCUSSION TITLE
Space between paragraphs

### DISCUSSION THREAD

**Discussion by antoniofsm** [uxbox/uxbox#548] [Q&A] [answered]
I'm sure more text format options are coming to Penpot. Now I'm looking for an option to add space between paragraphs (without adding blank lines or creating one object per paragraph) in the design panel or adding some CSS. This example comes from UXPin.
![imatge](https://user-images.githubusercontent.com/820272/106753803-74da7480-662c-11eb-88a9-5bf48399790a.png)

**Comment by elhombretecla** [comment, marked as answer]
Hi Antonio! Thanks for your feedback :)

As you said, a future text tool rework is on our road map. I'm going to add your idea to the research process to have it in mind.

**Reply by antoniofsm** [reply]
Great! Margin/padding in paragraphs is very usual in web design. So simple with CSS but an absolute hell in design tools! 😅
